src/features/transfer-ownership/FormSteps/Transaction.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+
+import { Wizard } from '@zero-tech/zui/components/Wizard';
+
+import { StepProps } from './types';
+
+import { StepText } from '../TransferOwnership.constants';
+
+export const Transaction: FC<StepProps> = ({ isTransferring }) => {
+	const transactionMessage = isTransferring
+		? 'Your transaction is being processed...'
+		: 'Please accept wallet transaction..';
+
+	return (
+		<>
+			<Wizard.Loading
+				message={
+					<div>
+						<p>{StepText.TRANSACTION_ALERT}</p>
+						<p>{transactionMessage}</p>
+					</div>
+				}
+			/>
+		</>
```

Comment:
Mentioned above, but you don't need a Fragment unless you're trying to return multiple elements. React requires you to return a single tree. 

Some more info here:
https://reactjs.org/docs/fragments.html

I'm guessing this will probably pop up a few times in this PR so I'll just leave the feedback on this one!

Reply:
Indeed there were a few to remove, missed them in refactoring. Updated 👍